__tests__/app.test.js

```diff
@@ -40,11 +46,40 @@ describe("/api", () => {
       .get("/api")
       .expect(200)
       .then((response) => {
-        console.log(response.body);
         const endpoints = response.body;
         expect(endpoints).toHaveProperty("GET /api");
         expect(endpoints).toHaveProperty("GET /api/topics");
         expect(endpoints).toHaveProperty("GET /api/articles");
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 Responds with: an article object, which should have the following properties: author, title, article_id, body, topic, created_at, votes, article_img_url", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        const article = response.body[0];
+        expect(article).toHaveProperty("author");
+        expect(article).toHaveProperty("title");
+        expect(article).toHaveProperty("article_id");
+        expect(article).toHaveProperty("body");
+        expect(article).toHaveProperty("topic");
+        expect(article).toHaveProperty("created_at");
+        expect(article).toHaveProperty("votes");
+        expect(article).toHaveProperty("article_img_url");
```

Comment:
Since we are only testing one article here we should test that the value of these properties are also what we are expecting.

Using toHaveProperty isn't bad, but there is better Jest Matchers we can use for more robust and extendable testing, take a look into toMatchObjects and expect.objectContaining.